### PR TITLE
Add Google tag

### PR DESCRIPTION
**Pull request by patmellon**
This PR adds a Global Site Tag to the site.

### CODE REVIEW

**Review by JoelMarcey — APPROVED** [review]

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -23,5 +23,17 @@
...
+
+<script async src="https://www.googletagmanager.com/gtag/js?id=UA-117752657-2"></script>
+
+<script>
+  window.dataLayer = window.dataLayer || [];
+
+  function gtag(){dataLayer.push(arguments);}
+
+  gtag('js', new Date());
+  gtag('config', 'UA-117752657-2');
```
Comment (JoelMarcey) [inline docs/source/_templates/layout.html]: What is the UA for? It is different for the ones I have for PyTorch.

**Review by JoelMarcey — COMMENTED** [review]
One question about the UA.

File: docs/source/_templates/layout.html (hunk 1 above)
Comment (Pat878) [reply docs/source/_templates/layout.html]: It's a Global Site Tag that @brucejlin asked me to add to the static, docs, and tutorial sites.

**Comment by zou3519** [comment]
@JoelMarcey is this good to go?

**Comment by JoelMarcey** [comment]
@zou3519 Yep. This can be merged.

**Pull request closed without merging**